src/main/resources/interface-specification.yml

```diff
@@ -640,11 +640,17 @@ components:
           type: string
         riskAnalysisForm:
           $ref: '#/components/schemas/RiskAnalysisFormSeed'
+        isFreeOfCharge:
+          type: boolean
+          default: true
```

Comment:
open to discussion, but default value could be placed in the process layer along with constraints